recipes/deepbgc/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "deepbgc" %}
+{% set version = "0.1.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "57946da7fe55b0a82b1d9b8737769db62c4fad4bc6ca75766442d36f63086d2d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
```

Comment:
This is OK, but it doesn't save any space or make it easier to diff. It's usually easier to just put `sha256 = "57946da7fe55b0a82b1d9b8737769db62c4fad4bc6ca75766442d36f63086d2d"` and be done with it.